airflow/www/app.py

```diff
@@ -88,6 +88,8 @@ def create_app(config=None):
             models.Connection, Session, name="Connections", category="Admin"))
         av(vs.VariableView(
             models.Variable, Session, name="Variables", category="Admin"))
+        av(vs.XComView(
```

Comment:
@bolkedebruin can u plz guide me how to achieve that.. as we don't use any auth  mechanism, I thought adding that into `Admin` category should do the trick.